dhcpv4/client4/client.go

```diff
@@ -88,7 +89,7 @@ func makeRawSocket(ifname string) (int, error) {
 	if err != nil {
 		return fd, err
 	}
-	err = BindToInterface(fd, ifname)
+	err = dhcpv4.BindToInterface(fd, ifname)
```

Comment:
Is the `BindToInterface` going to stay in the `dhcpv4` package?

Reply:
yes, for now at least, unless we have a better option. I thought of moving it to `client4` but it's also required for `server4`, and it feels awkward to call `client4.BindToInterface` from the server package

Reply:
I only see it being called from the client code. Is the plan to start using it for the server as well?

Reply:
As discussed offline: the `server` code will also use it. This will be shared by `server` and `client` so leaving it in the main directory for now.